content/02.introduction.md

```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
-Distant supervision has also been used in this domain [@tag:deep_dive], and in fact this effort was one of the motivating rationales for our work.
+Relationship extraction is the process of detecting semantic relationships from a collection of text.
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and heuristics for relationship extraction, (2) the use of unsupervised methods such as co-occurrence scores or clustering to find patterns within sentences and documents documents, and (3) the use of supervised or semi-supervised machine learning for classifying the presence of a relation within documents or sentences.
+In this section, we briefly discuss selected efforts under each category.
+
+#### Rule Based Extractors
+
+Rule based extractors rely heavily on expert knowledge to perform extraction.
+Typically, these systems use linguistic rules and heuristics to identify key sentences or phrases.
+For example, a hypothetical extractor focused on protein phosphorylation events would identify sentences containing the phrase "gene X phosphorylates gene Y" [@doi:10.1109/TCBB.2014.2372765].
+This word is a straightforward indication that two genes have a fundamental role in protein phosphorylation.
+Other phrase extractors have been used to identify drug-disease treatments [@tag:ctd_medline], pharmcogenomic events [@tag:pharmpresso] and protein-protein interactions [@tag:ppinterfinder; @tag:hpiminer].
+These extractors provide a simple but effective way to extract sentences; however, they depend on extensive knowledge about the text to be properly constructed.
+
+Besides using key phrases, a sentence's grammatical structure can be utilized to preform relationship extraction.
+Extractors under this approach involve the use of dependency trees.
+These trees are data structures that depict a sentence's grammatical relation structure in the form of nodes and edges.
+Nodes represent words and the edges represent the dependency type each word shares between one another.
+For example, a possible extractor would classify sentences as a positive if a sentence contained the following dependency tree path: "gene X (subject)-> promotes (verb)<- cell death (direct object) <- in (preposition) <-tumors (object of preposition)" [@tag:pkde4j].
+This approach provide extremely precise results, but the quantity of positive results remains modest as sentences appear in distinct forms and structure.
+Because of this limitation, recent approaches have incorporated methods on top of rule based extractors such as co-occurrence and machine learning systems [@doi:10.1186/s12859-018-2103-8; @tag:limtox].
+We discuss the pros and cons of added methods in a later section.
+For this project, we took a blind approach to construct our label functions; however, the discussed approaches in this section provides substantial inspiration for novel label functions in future endeavors.
```

Comment:
What does blind approach mean here?

Reply:
Ah I meant "blind approach" as in I constructed label functions without using any rules mentioned in discussed approaches.

Reply:
Changing for clarity